lib/normalize.js

```diff
@@ -80,7 +85,7 @@ module.exports = function normalize(path) {
 			result.push(part);
 		}
 	}
-	if(result.length === 1 && /^[A-Za-z]:$/.test(result))
+	if(result.length === 1 && /^[A-Za-z]:$/.test(result[0]))
```

Comment:
I fixed this because passing an array as the argument to test didn't make sense, though in fact it would work since the one element array conversion to string would just equal the string.  As far as adding a test case, I can't figure out how to trigger that if statement.  None of the existing tests do, and perhaps it is not possible.